.travis.yml

```diff
@@ -42,7 +42,7 @@ install:
   - 'bower update'
 
 before_script:
-  - grunt # this does syntax checking, unit testing, and jshint
+  - grunt --force # this does syntax checking, unit testing, and jshint
```

Comment:
Not sure how to do this without force. It always says 'Toc' not initialized, but that's how the documentation does it. https://afeld.github.io/bootstrap-toc/#via-javascript